.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+tracking.txt
+jobs.db
+acmOpportunities/jobs.db
```

Comment:
Remove the unused ignored files that your program no longer utilize.
```suggestion

```

Reply:
Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/ab61bc4ad59cbcf21b8df00d42f1b8e9a9bbd854